packages/cozy-harvest-lib/src/assets/datatypes/icon-picture.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M13 0h13v17a3 3 0 0 1-3 3H0V0h32v26H0V0h13z" id="a"/><path d="M13 0h13v17a3 3 0 0 1-3 3H0V0h32v26H0V0h13z" id="c"/></defs><g fill="none" fill-rule="evenodd"><g transform="translate(8 11)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><rect fill="#8EE39B" mask="url(#b)" width="32" height="26" rx="2"/><path d="M0 20l6.29-6.29a.999.999 0 0 1 1.416-.004L11 17l8.294-8.294a1.003 1.003 0 0 1 1.412 0L32 20v4.002C32 25.106 31.11 26 29.998 26H2.002A2.002 2.002 0 0 1 0 24.002V20z" fill="#1EC737" mask="url(#b)"/></g><g transform="translate(16 19)"><mask id="d" fill="#fff"><use xlink:href="#c"/></mask><rect fill="#8EE39B" mask="url(#d)" width="32" height="26" rx="2"/><path d="M0 20l6.29-6.29a.999.999 0 0 1 1.416-.004L11 17l8.294-8.294a1.003 1.003 0 0 1 1.412 0L32 20v4.002C32 25.106 31.11 26 29.998 26H2.002A2.002 2.002 0 0 1 0 24.002V20z" fill="#1EC737" mask="url(#d)"/></g><g transform="translate(0 3)"><rect fill="#8EE39B" width="32" height="26" rx="2"/><path d="M0 20l6.29-6.29a.999.999 0 0 1 1.416-.004L11 17l8.294-8.294a1.003 1.003 0 0 1 1.412 0L32 20v4.002C32 25.106 31.11 26 29.998 26H2.002A2.002 2.002 0 0 1 0 24.002V20z" fill="#1EC737"/><circle fill="#FFF" cx="8" cy="7" r="3"/><path d="M11 16l-5.5 5.5L11 16z" stroke="#8EE39B" stroke-linecap="round" stroke-linejoin="round"/></g></g></svg>
```

Comment:
this one is not optimized I think

Reply:
We're now using an svg inliner for babel that automatically optimize the svg at transpilation time.